### ISSUE TITLE
Upgrading from 0.0.13 to 0.1.4 breaks vite

### ISSUE THREAD

**Issue by mgarf** [HubSpot/calling-extensions-sdk#131]
**Checklist**
- [x] I have read the [FAQs](https://github.com/HubSpot/calling-extensions-sdk#faqs).
- [x] I have checked the [issue tracker](https://github.com/HubSpot/calling-extensions-sdk/issues) for duplicate issues.

**Description**

When i upgraded from 0.0.13 to 0.1.4 my UI could no longer build and the following error occurs
```
Failed to resolve entry for package "@hubspot/calling-extensions-sdk". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-scan]

    node_modules/esbuild/lib/main.js:1360:21:
      1360 │         let result = await callback({
           ╵                      ^
```
imports used:

`import * as CallingExtensions from "@hubspot/calling-extensions-sdk";`

and

`import CallingExtensions from "@hubspot/calling-extensions-sdk";`

**Comment by esme** [comment]
Hello, @mgarf! I apologize for any inconvenience this error may have caused. We are currently investigating the compatibility of v0.1.4 with Vite. While we look for a fix, you can utilize the following workaround:
```
import CallingExtensions from "@hubspot/calling-extensions-sdk/dist/main.js";
```

**Comment by esme** [comment]
We've released a patch in v0.1.5 that should fix this issue.

**Comment by alonso-cadenas** [comment]
Closing this bug since we've released a fix in v0.1.5. @mgarf, please feel free to reopen if the issue persists in your project!
